.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
```

Comment:
Casi que 100% seguro que CRA ya tiene linter configurado out of the box, con esto lo que estás haciendo es extendiendo el config 
https://create-react-app.dev/docs/setting-up-your-editor/#experimental-extending-the-eslint-config

Reply:
Mmm ok voy a revisar